test-scenarios/ovn-etcd-low-scale.yml

```diff
@@ -0,0 +1,36 @@
+global:
+  log_cmds: False
+
+cluster:
+  cluster_cmd_path: /home/mmahmoud/ovn-heater/runtime/ovn-fake-multinode
```

Comment:
@dceara I need the above setting w/o containers won't start and in the same time I can't leave like that what are you suggestions ?

Reply:
@msherif1234 Sure, but that can be a local change, in your repo.  These files are "samples" of test scenarios and should probably be as generic as possible.  Ideally we should get rid of `cluster_cmd_path` completely but that's a longer story and is not relevant for this PR.